src/deluge/io/midi/device_specific/midi_device_lumi_keys.h

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (c) 2023 Aria Burrell (litui)
+ *
+ * This file is part of The Synthstrom Audible Deluge Firmware.
+ *
+ * The Synthstrom Audible Deluge Firmware is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "io/midi/midi_device.h"
+
+#define MIDI_DEVICE_LUMI_KEYS_VP_COUNT 1
+
+#define MIDI_DEVICE_LUMI_KEYS_SYSEX_START 0xF0
```

Comment:
This block is generic - we probably have them defined in the existing sysex handling code already

Reply:
Fair enough, though it's also precompiler so shouldn't take much memory if any to define.

Reply:
Just checked. For as often as they're used there actually isn't an existing #define in place for them.

Reply:
Huh interesting. This was more of a clarity comment than a code size one but I'm surprised they aren't defined already